professionalism/building-confidence.md

```diff
@@ -10,7 +10,16 @@ N/A
 
 ### Motivation
 
-Confidence is an important variable to performance, achievement, and overall mental health and happiness! Achieving self-confidence is a tough barrier, especially in a competitive industry, and so it's important to understand the root causes of low self-confidence, and tools to build confidence.
+Confidence is an important variable to performance, achievement, and overall mental health and happiness! Achieving self-confidence is a tough barrier, especially in a competitive industry, and so it's important to understand the root causes of low self-confidence, and tools to build confidence. Your self- confidence is the key to your success. It is very important to know how to boost up your confidence so that you can make your own position in this competitive world.
+
+Its importance is justified in recent research, which shows that when people are put in situations where they are expected to not do well, their performance plunges. Quite literally, they get into a negative headspace and behave in an undesirable manner. However, when they’re expected to do well, they’re generally in a positive headspace and their performance improves considerably. Same person, different expectations.
+
+**Why do we need it in workspace?**
+
+- You’ll be more assertive: If your words and actions have conviction you’ll be taken more seriously, which will help you advance your job and career.
+- You’ll do more: You’ll be more likely to engage in challenging, but manageable projects. This will push the boundaries of your comfort zone, which will encourage you to aim for and achieve new goals. These are both valued of characteristics of successful workers. Most importantly, employers will learn to trust you with a project and know that you are likely going to be good at motivating others as well.
+- You’ll communicate more effectively: Confidence allows you to speak concisely and with clarity. Professionals who communicate with confidence can convey what they want to their clients and co-workers in a clear and efficient manner. Effective communication is critically important for career advancement.
+[(from kaplanprofessional.edu)](https://www.kaplanprofessional.edu.au/blog/why-is-confidence-in-the-workplace-important-and-how-do-i-improve-mine/)
```

Comment:
You could just have a link with (kaplanprofessional.edu).  Did you get all the new text in this file from the same site? You should add it to the ends of each section ( lines 16 and 23) on a new line without a bullet.